packer/README.md

```diff
@@ -8,12 +8,17 @@ homebrew:
 
 or retrieving it from the download available at https://www.packer.io/downloads.html
 
-## To Run
+## To build locally
 
-Set up your ATLAS_USER and ATLAS_TOKEN environment variables with your username
-and API access token from Atlas, then invoke:
+    packer build -only=virtualbox-iso template.json
+    packer push -name=$ATLAS_USERNAME/rackhd template.json
```

Comment:
Did you mean to remove this `packer push` line here since you included it below?